bam/src/ba.cc

```diff
@@ -780,16 +817,17 @@ void ba::_unapply_impact(kpi* kpi_ptr, ba::impact_info& impact) {
     _acknowledgement_soft -= impact.soft_impact.get_acknowledgement();
     _downtime_hard -= impact.hard_impact.get_downtime();
     _downtime_soft -= impact.soft_impact.get_downtime();
+    if (_dt_behaviour == configuration::ba::dt_ignore_kpi && impact.in_downtime)
+      return;
```

Comment:
if you "return" now, how the _level_hard and _level_soft values will increase on the next two lines ?